app/controllers/server.rb

```diff
@@ -1,9 +1,108 @@
+require_relative '../models/client'
+require_relative '../models/processor'
```

Comment:
no need to require other models manually. they are all required through this line in your `environment.rb`:

```ruby
# require the model(s)
Dir[APP_ROOT.join('app', 'models', '*.rb')].each { |file| require file }
```